src/main/scala/temple/generate/database/PostgresGenerator.scala

```diff
@@ -74,16 +74,22 @@ object PostgresGenerator extends DatabaseGenerator {
             .mkString(",\n")
             .pipe(indent(_))
         s"CREATE TABLE $tableName (\n$stringColumns\n);"
-      case Read(tableName, columns, condition) =>
+      case Read(tableName, columns, conditions) =>
         val stringColumns = columns.map(_.name).mkString(", ")
-        val stringCondition = condition.map(generateCondition) match {
-          case Some(conditionString) => List(s"WHERE $conditionString")
-          case None                  => List()
+        val stringConditions = conditions.map(generateCondition) match {
+          case Some(conditionsString) => List(s"WHERE $conditionsString")
+          case None                   => List()
```

Comment:
This is fine, but in the future try and avoid making changes to code that doesn't directly relate to the PR

Reply:
I know what you mean, but this does kinda relate to the PR. Had to write the same stuff for the Delete case and noticed it was wrong.